zndraw/server/events.py

```diff
@@ -340,21 +365,43 @@ def scene_pause(data):
 @io.on("modifier:register")
 def modifier_register(data):
     data["token"] = session["token"]
+    if "modifier" not in app.config["PER-TOKEN-DATA"][session["token"]]:
+        # this is a dict for the pyclient, but it has to be for every webclient ...
+        app.config["PER-TOKEN-DATA"][session["token"]]["modifier"] = {}
 
     try:
         # we can only register one modifier at a time
         name = data["modifiers"][0]["name"]
-        if name in app.config["MODIFIER"]:
-            # issue with the same modifier name on different webclients / tokens!
-            # only for default we need to ensure, there is only one.
-            raise ValueError(f"Modifier {name} is already registered.")
-        app.config["MODIFIER"][name] = request.sid
+        if name in app.config["MODIFIER"]["default_schema"]:
+            msg = f"Modifier {name} is already registered (default)."
```

Comment:
I think we should be more explicit here. "'{name}' is a default modifier and is reserved. Choose another name for your modifier!"